src/components/panel/panel.js

```diff
@@ -1320,6 +1377,7 @@ MdPanelRef.prototype._updatePosition = function(init) {
 
 /**
  * Focuses on the panel or the first focus target.
+ *
```

Comment:
So a lot of this extra space feels excessive, especially for methods like this one, where the description is only a single line and there's no extra information we care about. I've never seen it before (unless it's a multi-paragraph description).

Where does this pattern come from?

Here's the style guide we follow:
https://google.github.io/styleguide/javascriptguide.xml?showone=Comments#Comments